build/includes/sdk.mk

```diff
@@ -171,7 +172,7 @@ run-sdk-conformance-test-go:
 	# run without feature flags
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=go GRPC_PORT=9001 HTTP_PORT=9101
```

Comment:
This should be:

```
$(MAKE) run-sdk-conformance-test SDK_FOLDER=go GRPC_PORT=9001 HTTP_PORT=9101 TESTS=$(DEFAULT_CONFORMANCE_TESTS),$(COUNTS_AND_LISTS_TESTS)
```

As it needs to be told to check the Counter and Lists tests, since it's now enabled!

Reply:
Sorry, this fix isn't working for the C# and rest tests. Could there be something missing to enable the CL feature for these two tests? Please guide.

Reply:
For C# it's because of these lines:

https://github.com/googleforgames/agones/blob/db847357163b7ca974553bdf536a2541624ff12b/test/sdk/csharp/Program.cs#L169
https://github.com/googleforgames/agones/blob/db847357163b7ca974553bdf536a2541624ff12b/test/sdk/csharp/Program.cs#L295

Which is picking up `DOCKER_RUN_ARGS="--net host -e AGONES_SDK_GRPC_PORT=9005 -e AGONES_SDK_HTTP_PORT=9105 -e FEATURE_GATES=''`

Even though the feature gate is enabled by default `{"ctlConf":{"GameServerName":"","PodNamespace":"","Address":"","IsLocal":false,"LocalFile":"","Delay":2,"Timeout":40,"Test":"ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch,reserve,getcounter,updatecounter,setcountcounter,setcapacitycounter,getlist,updatelist,addlistvalue,removelistvalue","TestSdkName":"csharp","KubeConfig":"","GracefulTermination":true,"GRPCPort":9005,"HTTPPort":9105,"LogLevel":""},"featureGates":"CountsAndLists=true\u0026DisableResyncOnSDKServer=true\u0026Example=false\u0026GKEAutopilotExtendedDurationPods=false\u0026PlayerAllocationFilter=false\u0026PlayerTracking=false","message":"Starting sdk sidecar","severity":"info","source":"main","time":"2024-04-25T22:01:36.795794579Z","version":"1.41.0-dev-d0a74b1"}`

Reply:
Right now I'm thinking the easiest solution would be to create a `BETA_FEATURE_GATES` similar to the https://github.com/googleforgames/agones/blob/db847357163b7ca974553bdf536a2541624ff12b/build/Makefile#L67. Otherwise we'll need some way for C# to access "agones.dev/agones/pkg/util/runtime" similar to Go `runtime.FeatureEnabled(runtime.FeatureCountsAndLists)`.

Reply:
Would you mind submitting a PR to fix these two tests, please?

Reply:
Added a commit. 

A note: The Go SDK can see the runtime state of the SDK config, by using `runtime.FeatureEnabled(runtime.FeatureCountsAndLists)`. That means that for the first Go sdk conformance test Beta is on and Alpha is off, and for the second Go sdk conformance test both Alpha and Beta are on.

For REST and C# they can't see the runtime config, and depend on an environment variable being passed through. This means that for the first C# and REST tests Beta is on and Alpha is off, and for the second test Beta is off and Alpha is on.